packages/atomic/src/components/recommendations/atomic-recs-list/atomic-recs-list.tsx

```diff
@@ -100,6 +125,21 @@ export class AtomicRecsList implements InitializableComponent<RecsBindings> {
    */
   @Prop({reflect: true}) public headingLevel = 0;
 
+  @Watch('numberOfPages')
+  async pageChanged() {
```

Comment:
Based on the name of this method, I would expect it to be called when `currentPage` is changed.

Instead, I think this should be `onNumberOfPagesChanged` (or `onRecommendationsPerPageChanged`).